medical_emr/models/medical_disease_group_members.py

```diff
@@ -21,15 +21,14 @@
 #
 ###############################################################################
 
-from openerp.osv import fields, orm
+from openerp import fields, models
 
 
-class MedicalDiseaseGroupMembers(orm.Model):
+class MedicalDiseaseGroupMembers(models.Model):
```

Comment:
This model implements a M2M !
Without any other added value, it would be better to replace it with 2 m2m fields on medical.pathology.group and medical.pathology